BDKSwiftExampleWallet/View/Activity/LocalOutputItemView.swift

```diff
@@ -33,7 +33,7 @@ struct LocalOutputItemView: View {
 
             VStack(alignment: .leading, spacing: 5) {
 
-                Text(output.outpoint.txid)
+                Text(output.outpoint.txid.description)
```

Comment:
https://github.com/bitcoindevkit/bdk-ffi/commit/23d14f05e07276e1c5bce47bd453d2a0f8b8763d

Outpoint uses txid which is a type not a string now